lib/node_modules/@stdlib/array/base/join/test/test.js

```diff
@@ -0,0 +1,121 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var AccessorArray = require( '@stdlib/array/base/accessor' );
+var Float64Array = require( '@stdlib/array/float64' );
+var Int32Array = require( '@stdlib/array/int32' );
+var isString = require('@stdlib/assert/is-string');
+var join = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof join, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function joins an array-like object (generic)', function test( t ) {
+	var expected;
+	var actual;
+	var x;
+
+	x = [ 1, 2, 3 ];
+
+	expected = '1,2,3';
+	actual = join( x, ',' );
+
+	t.strictEqual( isString( actual ), true, 'returns expected value' );
+	t.deepEqual( actual, expected, 'returns expected value' );
```

Comment:
```suggestion
	t.strictEqual( actual, expected, 'returns expected value' );
```

`deepEqual` is inappropriate here. `strictEqual` does value and type equality.

This should be fixed here and elsewhere in this file.